hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1261,6 +1261,20 @@ The switch to turn S3A auditing on or off.
   </description>
 </property>
 
+<!--
+The switch to control how S3A handles glacier storage classes.
+-->
+<property>
+<name>fs.s3a.glacier.read.restored.objects</name>
+<value>READ_ALL</value>
+<description>
+  The config can have 3 values:
+
+  * READ_ALL: This would conform to the current default behavior of not taking into account the storage classes retrieved from S3. This will be done to keep the current behavior (i.e failing for an unrestored glacier class file) for the customers and not changing the experience for them.
```

Comment:
similar to above, remove This will be done to keep the current behavior for the customers and not changing the experience for them. and add something like "Retrieval of Galcier files will fail with xxx ` whatever the error is currently